File: 05_Open_source_ProjectRepository/AIAgentFramework/EnlightenmentLighthouseRuntime/micro_model/test_server.py
import http.server
import socketserver

PORT = 9002

class MyHTTPRequestHandler(http.server.SimpleHTTPRequestHandler):
    def do_GET(self):
        if self.path == '/health':
            self.send_response(200)
            self.send_header('Content-type', 'application/json')
            self.end_headers()
            self.wfile.write(b'{"status":"ok","service":"python-test-server"}')
        else:
            super().do_GET()

with socketserver.TCPServer(("127.0.0.1", PORT), MyHTTPRequestHandler) as httpd:
    print(f"Python test server started on 127.0.0.1:{PORT}")
    httpd.serve_forever()

File: 05_Open_source_ProjectRepository/AIAgentFramework/EnlightenmentLighthouseRuntime/micro_model/test_server.go
package main

import (
	"fmt"
	"net/http"
)

func main() {
	http.HandleFunc("/health", func(w http.ResponseWriter, r *http.Request) {
		w.Header().Set("Content-Type", "application/json")
		w.Write([]byte(`{"status":"ok","service":"test-server"}`))
	})

	fmt.Println("Test server started on 127.0.0.1:9002")
	if err := http.ListenAndServe("127.0.0.1:9002", nil); err != nil {
		fmt.Printf("Failed to start test server: %v\n", err)
	}
}
